packages/create-cloudflare/src/__tests__/templates.test.ts

```diff
@@ -318,3 +320,17 @@ describe("deriveCorrelatedArgs", () => {
 		);
 	});
 });
+
+describe("addCloudflareSystemPrompt", () => {
+	test("it should add the system prompt to the project", async () => {
+		const ctx = createTestContext();
+		await addCloudflareSystemPrompt(ctx);
+		// list all files in the project
+
+		const files = readdirSync(ctx.project.path, { recursive: true });
+		console.log(files); // this comes up as undefined?
```

Comment:
I don't know why this fails (tbh I don't know what kind of test to write here). can someone help me?